public/client/search.js

```diff
@@ -1,11 +1,28 @@
 // request button- activate modal
-function requestBook(bookId, pointCost, title){
+function requestBook(bookId, pointCost, title, userPoints){
     var modalBody = document.getElementById('modalBody');
-    modalBody.innerHTML = `Are you sure you want this book?: <br> <b>"${title}"</b>`
-    var confirmButton = document.getElementById('confirmButton');
-    confirmButton.setAttribute("onclick", `request(${bookId}, ${pointCost}, '${title}')`);
-    // toogle modal
-    $("#pageModal").modal("toggle");
+    if(pointCost > userPoints){
+        modalBody.innerHTML = `<b>You do not have enough points.</b><br>
```

Comment:
I'd recommend just having an actual modal on the page for this versus creating the modal content in the javascript. Creating it in javascript just makes it less clear when you're looking at the html and when using bootstrap should only be done if we're trying to make something dynamic about the content. Since this is static content, your function will just do the point check and then display the modal if needed. For the confirm button, since that'll be part of your modal in the html just add an onclick there rather than setting it in the function.

Also for the sake of consistency in the application, we should do all modals similar. In every other case we have modal content in the html unless there is dynamic content populated by a function call.

Reply:
I have applied this suggestion.